af/fastpath/fastpath/core.py

```diff
@@ -377,6 +377,25 @@ def match_fingerprints(measurement):
     return matches
 
 
+def g_or(d, key, default):
+    """Dict getter: return 'default' if the key is missing or value is None"""
+    # Note: 'd.get(key) or default' returns 'default' on any falsy value
+    v = d.get(key)
+    return default if v is None else v
+
+
+def g(d, *keys, default=None):
```

Comment:
Maybe you should add a check to ensure len(keys) > 0, otherwise you will get an IndexError.

Reply:
If we forget to add  the keys throwing an IndexError would be ok.